### ISSUE TITLE
Infinite unproductive output on empty credentials

### ISSUE THREAD

**Issue by yutv** [akeneo/magento2-connector-community#178]
### Environment and configuration

1. Magento Cloud 2.3.4
2. akeneo/module-magento2-connector-community=100.4.2 

### Steps to reproduce
1. Leave blank fields (Akeneo username, Akeneo password, Akeneo API client ID, Akeneo API client secret) in the **Stores > Configuration > Catalog > Akeneo Connector > Akeneo API Configuration**
1. run the following command 
```
php bin/magento akeneo_connector:import --code=category,family,attribute,option,family_variant,product
```
### Expected result

- error message about empty Akeneo credentials 
- exit

### Actual result

Infinite loop with the following output
```
[18:09:55] Start import
[18:09:55] completed
[18:09:55] Start import
[18:09:55] completed
[18:09:55] Start import
[18:09:55] completed
[18:09:55] Start import
[18:09:55] completed
[18:09:55] Start import
[18:09:55] completed
[18:09:55] Start import
[18:09:55] completed
[18:09:55] Start import
[18:09:55] completed
[18:09:55] Start import
[18:09:55] completed
[18:09:55] Start import
[18:09:55] completed
[18:09:55] Start import
[18:09:55] completed
[18:09:55] Start import
[18:09:55] completed
[18:09:55] Start import
[18:09:55] completed
[18:09:55] Start import
[18:09:55] completed
[18:09:55] Start import
[18:09:55] completed
[18:09:55] Start import
```

**Comment by yutv** [comment]
Is there any progress with this issue? It is still unresolved even in the v100.4.11.

**Comment by krasii** [comment]
The same issue

**Comment by DGPRoman** [comment]
did anyone solve this?

**Comment by Dnd-Gimix** [comment]
Hello @yutv @krasii @DGPRoman ,

Thank you for your interest in Akeneo Connector for Magento 2.
We have released a bug fix for this in the latest 100.4.12 version of the connector.

Could you try again and tell us if it is now OK for the credentials check ?

Many thanks.

Regards,
